modules/virt-defining-apps-for-dr.adoc

```diff
@@ -0,0 +1,57 @@
+
+Define applications for disaster recovery by using either {rh-rhacm-first}-managed VMs or RHACM-discovered VMs.
+
+An RHACM-managed application that includes a VM must be created by using a GitOps workflow and by creating an RHACM Application or `ApplicationSet`.
+
+You can configure any other VM in the cluster as an RHACM-discovered application. This includes VMs imported by using the Migration Toolkit for Virtualization (MTV), VMs created by using the {VirtProductName} web console, or VMs created by any other means, such as the CLI. Applications in an RHACM-discovered VM are discovered by the VM through user-defined labels.
+
+[id="best-practices-rhacm-managed-vm_{context}"]
+== Best practices when defining an RHACM-managed VM
+
+There are several actions you can take to improve your experience and chance of success when defining an RHACM-managed VM.
+
+[discrete]
+[id="use-a-pvc-and-populator_{context}"]
+=== Use a PVC and populator to define storage for the VM
+Because data volumes create persistent volume claims (PVCs) implicitly, data volumes and VMs with data volume templates do not fit as neatly into the GitOps model.
+
+[discrete]
+[id="use-import-method_{context}"]
+=== Use the import method when choosing a population source for your VM disk
+This method works around limitations in Regional-Dr that prevent you from protecting VMs that use cloned PVCs.
+
+Select a RHEL image from the software catalog to use the import method. It is recommended to use a specific version of the image rather than a floating tag for consistent results. The KubeVirt community maintains container disks for other operating systems in a Quay repository.
+
+[discrete]
+[id="use-pull-node_{context}"]
+=== Use `pullMethod: node`
+Use the pod `pullMethod: node` when creating a data volume from a registry source to take advantage of the {product-title} pull secret, which is required to pull container images from the Red Hat registry.
+
+[id="best-practices-rhacm-discovered-vm_{context}"]
+== Best practices when defining an RHACM-discovered virtual machine
+
+There are several actions you can take to improve your experience and chance of success when defining an RHACM-discovered VM.
+
+[discrete]
+[id="protect-the-vm_{context}"]
+=== Protect the VM when using MTV, the {VirtProductName} web console, or a custom VM
+Because automatic labeling is not currently available, the application owner must manually label the components of the VM application when using MTV, the {VirtProductName} web console, or a custom VM.
+
+After creating the VM, apply a common label to the following resources associated with the VM: `VirtualMachine`, `DataVolume`, `PersistentVolumeClaim`, `Service`, `Route`, `Secret`, and `ConfigMap`. Do not label virtual machine instances (VMIs) or pods, since these are created and managed automatically by {VirtProductName}.
+
+[discrete]
+[id="working-vm-contains_{context}"]
+=== A working VM typically contains more than just the `VirtualMachine` object
```

Comment:
**issue**: All other headings are actions except this one and the following one "VMs may be part of a larger logical application". We should rewrite the headings to be actionable. Maybe Include additional data to. your VMs or something along the lines.